src/main/java/thePackmaster/cards/pinnaclepack/MarvelousFeastUncommonPower.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.pinnaclepack.Cooking;
+import thePackmaster.powers.pinnaclepack.Cooking_;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class MarvelousFeastUncommonPower extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("MarvelousFeastUncommonPower");
+    private boolean fixUpgradeCheck = false;
+
+    public MarvelousFeastUncommonPower() {
```

Comment:
*Marvelous Feast*
The design of this card has multiple issues.

This card uses multiple keywords when it should instead write out its effects -- it should be changed to clearly and directly describe what it does.. This card should also should preview the Croquette cards using MultiCardPreview (e.g. put `MultiCardPreview.add(this, new FishyCroquettesSpecialColourless(), new FriendshipCroquettesSpecialColourless(), etc);` in the constructor, with the croquettes ordered by the order they show up).

This means that the card will read something like: "At the end of the next four turns, add a *Croquette to your hand, in the order *Friendship, *Fishy, *Meaty, *Mystery." None of those need to be keywords, since the preview cards will show the croqeuttes.

Finally, the upgrade to this card goes against how upgrades normally work (by increasing the cost of the card; this is another thing that might be okay in a pack where it was the only odd thing, but in a pack with many odd things it's an issue). Instead of increasing cost, the upgrade should keep the same cost, and the upgraded Croquettes should be rebalanced as needed to fit that by having the upgrades increase numbers by less than the currently do.